File: invitationtracker/src/modules/database/src/main/java/tracker/database/UserInvite.java
package tracker.database;

import lombok.AllArgsConstructor;
import lombok.Value;

import java.sql.Timestamp;

@Value
@AllArgsConstructor
public class UserInvite
{
    String guildId;
    String targetChannelId;
    String joinedUserID;
    String usedInvite;
    String inviteIssuerId;
    Timestamp joinedAt;
    Timestamp inviteCreatedAt;
}

File: nginx/MASZ/src/app/models/UserInvite.ts
export interface UserInvite {
    id: number;
    guildId: string;
    targetChannelId?: string;
    joinedUserId: string;
    usedInvite: string;
    inviteIssuerId: string;
    joinedAt: Date;
    inviteCreatedAt: Date;
}